### ISSUE TITLE
Use sigmod or tempered_sigmoid for prediction?

### ISSUE THREAD

**Issue by jamesguoxin** [google/bi-tempered-loss#12]
Dear authors,

I'm encountering an issue regarding to the prediction phase. Assume we use bi_tempered_binary_logistic_loss as our loss function, should we still use sigmod as function used for probability calculation, or we should use tempered_sigmoid? Thanks for clarification!

**Comment by eamid** [comment]
You should use the `tempered_sigmoid` function to calculate the class probabilities. However, to predict the class, you don't need to form the probabilities (i.e. pick the class with the larger logit value).

**Comment by jamesguoxin** [comment]
> 

Hello eamid,

Thank you for your reply. In our case, we would like to extract the probability to calculate AUC for the prediction. So for the t parameter in tempered_sigmoid, we should keep it consistent with bi_tempered_binary_logistic_loss t1?

**Comment by eamid** [comment]
It should be consistent with `t2`. The other temperature `t1` controls the boundedness of the loss

**Comment by jamesguoxin** [comment]
I see, thank you eamid!
